content/en/user-manual/graphics/lighting/lightmapping.md

```diff
@@ -17,6 +17,10 @@ PlayCanvas offers two ways to use lightmaps in your scene: **External lightmap g
 
 This page gives details and best practices on rendering lightmaps from external tools.
 
+<div class="alert-info">
```

Comment:
This limitation does not apply to lightmaps baked in other tools .. as if those bake lightmaps to atlas as they normally do, batching would work. So I'd suggest to remove this part.

Reply:
When you say bake, do you mean as part of the diffuse or as a separate texture for the lightmap section of the material or either?
![image](https://user-images.githubusercontent.com/16639049/113509798-625ab980-954f-11eb-9188-ebfe8d14140e.png)

Reply:
either. External lightmaps either stored in diffuse or lightmap slot of materials work with batching - as material batching works. Internally baked lightmaps are not stored in material, but in meshInstances .. and that's the reason batching does not work. And the reason they're stored in meshInstances is that each meshInstance has different lightmap due to not having atlas, and so they are different textures. We need to implement lightmap atlasing which is on my list in the next few months.